### DISCUSSION TITLE
RectanglePro conflicting with other apps?

### DISCUSSION THREAD

**Discussion by ioVideoMaker** [rxhanson/RectanglePro-Community#456] [General]
Hello everyone!
I'm a videomaker and I also have some youtube channels, so I often record my screen for making tutorials.

I use both "Cursor Pro" and "Keystroke Pro" which seem to be conflicting with RectanglePro.

Specifically, I cannot longer see the "Snap Panel" (which is more intuitive to use, as newbie, IMHO).

Another problem here: I defined 3 "Custom Size & Position" but the icon in the snap panel is always the same, making very difficult for me to quickly assess when I'll be going to have my window placed (my working memory is overloaded with like a hundred keyboard shortcuts for NLE, Image Editors, etc. .

Thanks to anybody so smart to fix this!

ciao!
Marco (ioVideoMaker)

Used for years Divvy, Magnet and others, at first sight I think RectanglePro is more complete and could be the only app needed for screen space and windows optimisation!

**Comment by rxhanson** [comment]
Thanks for trying Rectangle Pro and for the report!

On the conflicting apps, can you share a quick screen recording so I can see what's going on?

> Another problem here: I defined 3 "Custom Size & Position" but the icon in the snap panel is always the same, making very difficult for me to quickly assess when I'll be going to have my window placed (my working memory is overloaded with like a hundred keyboard shortcuts for NLE, Image Editors, etc. .

If you give the custom size & position a name, an abbreviation will be displayed on the icon. It's on my backlog to improve this further.

**Comment by ioVideoMaker** [comment]
Hello Ryan, nice to meet you.

> Thanks for trying Rectangle Pro and for the report!
I'm not trying it, it's so damn good I'm buying it!

> On the conflicting apps, can you share a quick screen recording so I can
see what's going on?

please check here: https://mab.to/t/xCDjJnTp6Gz/eu1
(I use 3 displays so I needed an external recorder)

Please help me, Obi Wan Hanson, you're my only hope to get a single app for
quickly arranging my windows, even at defined res, while I record my
YouTube tutorials!!
I have faith in you!

have a wonderful day
Marco - ioVideoMaker








Il giorno gio 22 feb 2024 alle ore 06:04 Ryan Hanson <
***@***.***> ha scritto:

**Reply by rxhanson** [reply]
Sorry for the delay here, and thanks for the video. 

It might be worth a try to adjust the way that Rectangle Pro obtains the mouse under the cursor. Can you try executing this Terminal command, then restart the app?

```
defaults write com.knollsoft.Hookshot systemWideMouseDown -int 1
```

Let me know how if that improves the situation for either Cursor Pro or Keystroke Pro.

**Comment by ioVideoMaker** [comment]
Hi Ryan, it worked!!!

Very Good!
Now I can make an even more interesting video, showing before and after!
Any clue about the release date for this new implementation?

have a nice day

ciao!
Marco

Il giorno mar 27 feb 2024 alle ore 06:42 Ryan Hanson <
***@***.***> ha scritto:
